draft-ietf-webtrans-http2.md

```diff
@@ -251,368 +285,353 @@ unidirectional, indicated by the second least significant bit of the
 to inform stream state transitions. Wherever QUIC relies on receiving an ack
 for a packet to transition between stream states, WebTransport performs that
 transition immediately.
 
-# WebTransport Frames
+# WebTransport Capsules
 
-WebTransport frames mirror their QUIC counterparts as closely as possible to
-enable maximal reuse of any applicable QUIC infrastructure by implementors.
+WebTransport capsules mirror their QUIC frame counterparts as closely as
+possible to enable maximal reuse of any applicable QUIC infrastructure by
+implementors.
 
-A WebTransport frame begins with a Frame Type and Frame Length which are
-followed by zero or more fields that are type-dependent.
+WebTransport capsules follow the Capsule Protocol format defined in {
+{Section 3.2 of HTTP-DATAGRAM}}, containing a Capsule Type and Capsule Length
+which are followed by zero or more bytes of Capsule Value that are
+type-dependent.
 
 ~~~
-Frame {
-  Frame Type (i),
-  Frame Length (i),
-  Type-Dependent Fields (..),
+Capsule {
+  Capsule Type (i),
+  Capsule Length (i),
+  Capsule Value (..),
 }
 ~~~
-{: #fig-wt_frame_header title="WebTransport Frame Format"}
+{: #fig-Capsule_Protocol_header title="Capsule Format"}
 
-The Frame Type field indicates the type of the frame, defining what
+The Capsule Type field indicates the type of the capsule, defining what
 type-dependent fields will be present.
 
-The Frame Length field indicates the length of the WebTransport frame, including
-all type-dependent fields and other information. It does not include the size
-of the Frame Type or Frame Length fields themselves.
+The Capsule Length field indicates the length of the WebTransport capsule,
+including all type-dependent fields and other information. It does not include
+the size of the Capsule Type or Capsule Length fields themselves.
 
-Both of these fields use a variable-length integer encoding (see {{Section 16 of
-!RFC9000}}), with one exception. To ensure simple and efficient
-implementations of frame parsing, the frame type and length MUST use the
-shortest possible encoding. For example, for the frame types defined in this
-document, this means a single-byte encoding, even though it is possible to
-encode these values as a two-, four-, or eight-byte variable-length integer.
+## PADDING Capsule {#PADDING}
 
-## WT_PADDING Frames {#WT_PADDING}
+*[PADDING]: #
 
-*[WT_PADDING]: #
-
-A WT_PADDING frame (type=0x00) has no semantic value. PADDING frames can be used
-to introduce additional data between other WebTransport frames and can also be
-used to provide protection against traffic analysis or for other reasons.
+A PADDING capsule is an HTTP capsule {{HTTP-DATAGRAM}} of type=0x00 and has no
+semantic value. PADDING capsules can be used to introduce additional data
+between other HTTP datagrams and can also be used to provide protection
+against traffic analysis or for other reasons.
 
 ~~~
-WT_PADDING Frame {
+PADDING Capsule {
```

Comment:
We need to pick new numbers for all of these

Reply:
Done, thank you! :)